packages/web/src/components/form-fields/TextAreaField.tsx

```diff
@@ -0,0 +1,16 @@
+import { useField } from 'formik'
+
+import { TextAreaV2, TextAreaV2Props } from 'components/data-entry/TextAreaV2'
+
+type TextAreaFieldProps = TextAreaV2Props & {
+  name: string
+}
+
+export const TextAreaField = (props: TextAreaFieldProps) => {
+  const { name, ...other } = props
+  const [field, meta] = useField(name)
+
+  const hasError = Boolean(meta.touched && meta.error)
+
+  return <TextAreaV2 {...field} error={hasError} {...other} />
```

Comment:
is the plan to render error message separately?

Reply:
yeah i havent determined yet if this should be the job of the input or the field... i think probably the input, what do you think?